ReadMe.md

```diff
@@ -90,9 +90,13 @@ English | Japanese | Kana | Notes
 --------|----------|------|------
 Development | 開発 | かいはつ |
 Developer | 開発者 | かいはつしゃ |
+Programmer | PG/エンジニア | エンジニア | 
+Engineer | PG | ぷろぐらまー |
```

Comment:
These two words seem very debatable. Would love some extra feedback on this from either @AlmirKadric or @endow.